docs/ebpf/users.md

```diff
@@ -4,10 +4,9 @@ Below is a non-comprehensive list of open-source software built with {{ proj }},
 just for inspiration or to gain a better understanding of how to tackle certain
 problems using eBPF.
 
-A live list of package importers can be found at [:fontawesome-brands-golang: {{
-proj }} package
-importers](https://pkg.go.dev/github.com/cilium/ebpf?tab=importedby). If you'd
-like to include a project on this page, feel free to open a pull request.
+A list of :fontawesome-brands-golang: {{ proj }} importers can be found on
+[Sourcegraph].
```

Comment:
Nit: this paragraph can be reflowed. I use https://marketplace.visualstudio.com/items?itemName=stkb.rewrap to do this while authoring.